src/main/java/com/ing/data/cassandra/jdbc/CassandraConnection.java

```diff
@@ -504,4 +533,25 @@ public String toString() {
         throw new SQLFeatureNotSupportedException(String.format(NO_INTERFACE, iface.getSimpleName()));
     }
 
+
+    public OptionSet getOptionSet() {
+        return optionSet;
+    }
+
+    private OptionSet lookupOptionSet(String property) {
+        ServiceLoader<OptionSet> loader = ServiceLoader
```

Comment:
The local variables and method parameters should be declared final as per our Checkstyle rules:
```
[WARN] /home/runner/work/cassandra-jdbc-wrapper/cassandra-jdbc-wrapper/src/main/java/com/ing/data/cassandra/jdbc/CassandraConnection.java:541:39: Parameter property should be final. [FinalParameters]
[WARN] /home/runner/work/cassandra-jdbc-wrapper/cassandra-jdbc-wrapper/src/main/java/com/ing/data/cassandra/jdbc/CassandraConnection.java:542:34: Variable 'loader' should be declared final. [FinalLocalVariable]
[WARN] /home/runner/work/cassandra-jdbc-wrapper/cassandra-jdbc-wrapper/src/main/java/com/ing/data/cassandra/jdbc/CassandraConnection.java:544:29: Variable 'iterator' should be declared final. [FinalLocalVariable]
[WARN] /home/runner/work/cassandra-jdbc-wrapper/cassandra-jdbc-wrapper/src/main/java/com/ing/data/cassandra/jdbc/CassandraConnection.java:546:23: Variable 'optionSet' should be declared final. [FinalLocalVariable]
[WARN] /home/runner/work/cassandra-jdbc-wrapper/cassandra-jdbc-wrapper/src/main/java/com/ing/data/cassandra/jdbc/CassandraConnection.java:552:19: Variable 'optionSet' should be declared final. [FinalLocalVariable]
```